ts/store/reducers/persistedPreferences.ts

```diff
@@ -215,8 +215,8 @@ export const isIdPayTestEnabledSelector = (state: GlobalState) =>
 export const isDesignSystemEnabledSelector = (state: GlobalState) =>
   state.persistedPreferences.isDesignSystemEnabled ?? false;
 
-export const isNewWalletSectionLocallyEnabledSelector = (state: GlobalState) =>
-  state.persistedPreferences?.isNewWalletSectionEnabled ?? false;
+export const isNewScanSectionLocallyEnabledSelector = (state: GlobalState) =>
+  !!state.persistedPreferences?.isNewScanSectionEnabled;
```

Comment:
In my opinion, a remote FF should also be inserted here to show the functionality, alternatively we should set this local FF to true before going into production, what do you think?
This is my proposal: 
```
export const isNewScanSectionLocallyEnabledSelector = (state: GlobalState) =>
  !!state.persistedPreferences?.isNewScanSectionEnabled || isRemoteFFActive
```

Reply:
The remote feature flag of this implementation is not the subject of this PR, what do you think about the implementation in another ad-hoc PR?

Reply:
Yes, it's a good idea!